lib/spacex/base_request.rb

```diff
@@ -15,5 +15,21 @@ def self.get(path)
       end
       Hashie::Mash.new(data.get.body)
     end
+
+    def self.getAll(path)
+      data = Faraday.new(
+        url: "#{SPACEX::ROOT_URI}/#{path}",
+        request: {
+          params_encoder: Faraday::FlatParamsEncoder
+        }
+      ) do |c|
+        c.use ::FaradayMiddleware::ParseJson
+        c.use Faraday::Response::RaiseError
+        c.use Faraday::Adapter::NetHttp
+      end
+      arr = []
+      data.get.body.each { |k| arr << k['id']}
+      return arr
```

Comment:
Up to you, but you can remove the return and just have a `.map { |k ...`.

`.map` automatically returns and have the same effect of `return arr`.